Exdi/exdigdbsrvsample/GdbSrvControllerLib/AsynchronousGdbSrvController.cpp

```diff
@@ -576,11 +586,20 @@ bool AsynchronousGdbSrvController::HandleInterruptTarget(_Inout_ AddressType * p
 
     bool isBreakDone = false;
     *pEventNotification = false;
+
+    // check of the asyc recv is still active
+    if (!IsAsynchronousCommandInProgress())
+    {
+        StartAsynchronousCommand("", true, false);

```

Comment:
Can you add a comment around what/why a blank command does to the target or gdbserver?